build/run-backup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Should we upload log of the backup to the bucket as Marco asked?

Reply:
I did in https://github.com/percona/percona-server-mysql-operator/pull/77/commits/bd62378bfee59f896ad46ef68238db0de4ff0b28 but reverted it https://github.com/percona/percona-server-mysql-operator/pull/77/commits/e8d4683fb01d4436402a55e2cbf57fc1f0635f9f. I thought we could do it with `xbcloud` but I was wrong, it doesn't support uploading anything other than backups. I created https://jira.percona.com/browse/K8SPS-89 for the next release.